__tests__/fixtures/upgrade/package-not-in-registry/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "yarn-test-git-repo": "https://github.com/juanca/yarn-test-git-repo.git#master"
+  }
```

Comment:
Would it be possible to have a branch for this test in https://github.com/yarnpkg/e2e-test-repo?
Just to make Yarn CI more reliable